rest/model/reliability.go

```diff
@@ -25,40 +25,40 @@ type APITaskReliability struct {
 }
 
 // Converts a service level struct to an API level struct
-func (apiTaskReliability *APITaskReliability) BuildFromService(h interface{}) error {
+func (tr *APITaskReliability) BuildFromService(h interface{}) error {
 	switch v := h.(type) {
 	case *reliability.TaskReliability:
-		apiTaskReliability.TaskName = utility.ToStringPtr(v.TaskName)
-		apiTaskReliability.BuildVariant = utility.ToStringPtr(v.BuildVariant)
-		apiTaskReliability.Distro = utility.ToStringPtr(v.Distro)
-		apiTaskReliability.Date = utility.ToStringPtr(v.Date.UTC().Format("2006-01-02"))
+		tr.TaskName = utility.ToStringPtr(v.TaskName)
+		tr.BuildVariant = utility.ToStringPtr(v.BuildVariant)
+		tr.Distro = utility.ToStringPtr(v.Distro)
+		tr.Date = utility.ToStringPtr(v.Date.UTC().Format("2006-01-02"))
 
-		apiTaskReliability.NumSuccess = v.NumSuccess
-		apiTaskReliability.NumFailed = v.NumFailed
-		apiTaskReliability.NumTotal = v.NumTotal
-		apiTaskReliability.NumTimeout = v.NumTimeout
-		apiTaskReliability.NumTestFailed = v.NumTestFailed
-		apiTaskReliability.NumSystemFailed = v.NumSystemFailed
-		apiTaskReliability.NumSetupFailed = v.NumSetupFailed
-		apiTaskReliability.AvgDurationSuccess = v.AvgDurationSuccess
-		apiTaskReliability.SuccessRate = v.SuccessRate
+		tr.NumSuccess = v.NumSuccess
+		tr.NumFailed = v.NumFailed
+		tr.NumTotal = v.NumTotal
+		tr.NumTimeout = v.NumTimeout
+		tr.NumTestFailed = v.NumTestFailed
+		tr.NumSystemFailed = v.NumSystemFailed
+		tr.NumSetupFailed = v.NumSetupFailed
+		tr.AvgDurationSuccess = v.AvgDurationSuccess
+		tr.SuccessRate = v.SuccessRate
 	default:
-		return errors.Errorf("incorrect type when converting task stats (%T)", v)
+		return errors.Errorf("programmatic error: expected task reliability stats but got type %T", h)
```

Comment:
Why `task reliability stats` and not just `task reliability`?

Reply:
Because they're statistics on tasks.